specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -237,6 +247,19 @@ batch:
   - tag: package-2020-02-18-preview
   - tag: package-2018-03-preview
 ```
+### Tag: package-2021-01-01 and java
```

Comment:
is it possible that we extract the java configuration to a standalone file `readme.java.md`? It looks kind of messy here

Reply:
@ArcturusZhang , I haven't done that before, would you please give me an example so to do that? Thanks

Reply:
Hi @zhaomuzhi it is OK if we do not do this in this PR. I could ask the owner of Java SDK to do this instead after this PR gets merged